.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,22 @@
+name: Semgrep
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/semgrep.yml
```

Comment:
```suggestion
```

Doesn't seem to feature in the [Semgrep sample CI config](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#github-actions) in their docs, even though present in their auto-generated code from the app.